lib/commands/init.js

```diff
@@ -29,6 +30,13 @@ module.exports = Command.extend({
     },
     { name: 'yarn', type: Boolean }, // no default means use yarn if the blueprint has a yarn.lock
     { name: 'name', type: String, default: '', aliases: ['n'] },
+    {
+      name: 'lang',
+      type: String,
+      default: '',
+      description: 'Sets the base human language of the application via index.html',
+      aliases: ['l'],
```

Comment:
Does it need a short alias?

Reply:
> Does it need a short alias?

That's a great point! Probably doesn't need one. We started out with the --language option (with -lang and -l as the aliases), but decided to make the option --lang so it directly matched the html attribute it was assigning. If -lang was short enough to be an alias before that change, makes sense that --lang doesn't need a short alias.

Reply:
Seems fine to keep, though I do want to point out that we should **not** use the shortened versions of _any_ flags in our own documentation or guides (other than to document the shorthand :wink:).

Reply:
I vote for removing it. --lang is already short, and we consume -l when we may want it for a future longer option starting with L.

Reply:
I don't think it's really my place to 'vote' here, but I did think about this throughout the RFC process from as many angles as possible:

Reasons to keep:
- no reason not to consume for a feature that was fairly well-received by the community
- an a11y-related flag is something you don't see often (if at all), so landing the feature has a bit of a 'making a statement' PR bonus to it. I don't know if that would be diminished by making `--lang` not as 'fully-featured' as other things
- there was definitely a great deal of 'how do we name this' discussion during review, but there wasn't much concern raised about the alias. However, I'm willing to admit that this might be due to there being more pressing concerns to discuss (e.g., confusion WRT programming languages) and a group bias (people working on an a11y-st are probably more likely to think the flag 'deserves' an alias)
- there is definitely a precedent for using two-to-three letter aliases as needed, and in principle, the future longer option starting with L would be more suitable to one of those anyway

Reasons to remove:
- don't want to be wasteful in general -- *especially* in `new`
- don't want to cause confusion with some other widely-used `-l` (here or elsewhere, things like `--list` and `--local` come to mind)
- don't want to consume the alias if there's a longer, more sensible choice coming in the roadmap
- the decision to go with `--lang` over `--language` was largely shaped by `lang` being the specific attribute set in `index.html` -- i.e., that it was an a11y-related teaching opportunity. I know that we do plan to teach this appropriately (and only with the full name) where the new flag is used in the documentation, but I suppose there's a chance for that opportunity to be lost / the meaning to be easily forgotten if we provide a shortcut

Taken together, I'd probably say that unless there's a concrete precedent that we will cause confusion with some other widely-used `-l` or evidence that we've got something in the roadmap that would definitely be a more suitable choice, it makes most sense to keep it.

Reply:
We can always add it later once the pain of the full option is known. It's however not as easy to remove it or repurpose it once a better use comes along.

Reply:
How do we determine which flags are "better" than another flag? Currently this RFC was accepted and is being implemented, which seems "better" or "more useful" than a hypothetical potential thing that may exist in the future. I for one vote to give this flag as much importance as any other flag that we have added in the past.